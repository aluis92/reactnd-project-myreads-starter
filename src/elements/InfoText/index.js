import React from 'react';
import { node } from 'prop-types';

import {
    Container,
    Text,
} from './styles';

function InfoText({
    children,
}) {
    return (
        <Container>
            <Text>{children}</Text>
        </Container>
    );
}

InfoText.propTypes = {
    children: node.isRequired,
};

export default InfoText;
