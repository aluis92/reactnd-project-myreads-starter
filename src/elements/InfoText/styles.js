import styled from 'styled-components';

export const Container = styled.header`
    padding: 50px;
`;

export const Text = styled.h1`
    text-align: center;
    font-size: 1.5em;
`;
