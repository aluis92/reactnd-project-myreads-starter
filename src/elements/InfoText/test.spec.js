import React from 'react';
import {
    shallow,
} from 'enzyme';

import InfoText from './index';

describe('InfoText component', () => {
    it('should render if it has a child', () => {
        expect(shallow(
            <InfoText>something</InfoText>,
        )).toMatchSnapshot();
    });
});
