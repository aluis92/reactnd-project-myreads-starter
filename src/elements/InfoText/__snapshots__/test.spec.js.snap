// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoText component should render if it has a child 1`] = `
<styled.header>
  <styled.p>
    something
  </styled.p>
</styled.header>
`;
